wled00/usermod.cpp

```diff
@@ -119,7 +119,7 @@ void userLoop() {
           sampleAvg = receivedPacket.sampleAvg;
           // VERIFY THAT THIS IS A COMPATIBLE PACKET
           char packetHeader[6];
-          memcpy(&receivedPacket, packetHeader, 6);
+          memcpy(packetHeader, &receivedPacket, 6);
           if (!(isValidUdpSyncVersion(packetHeader))) {
```

Comment:
Can you confirm that instructions in the next `if` block (lines 123 to 145) will work as expected? 

Because - if i understand the bug correctly - these were never executed previously?